azure-stack/hci/concepts/host-network-requirements.md

```diff
@@ -277,17 +277,24 @@ Stretched clusters provide disaster recovery that spans multiple data centers. I
 Stretched clusters have the following requirements and characteristics:
 
 - RDMA is limited to a single site, and is not supported across different sites or subnets.
+
 - Servers in the same site must reside in the same rack and Layer-2 boundary.
-- Communication between sites cross a Layer-3 boundary; stretched Layer-2 topologies are not supported.
 
-- If a site uses RDMA for its storage adapters, these adapters must be on a separate subnet and VLAN that cannot route between sites. This prevents Storage Replica from using RDMA across sites.
+- Communication between sites must cross a Layer-3 boundary; stretched Layer-2 topologies are not supported.
```

Comment:
While you're at it, this should probably say "Host communication between sites..."